src/components/comment-form/comment-form.stories.mdx

```diff
@@ -92,6 +96,10 @@ There is also a reply version of this component (`is_reply: true`), used to repl
 ## Template Properties
 
 - `class`: Append a class to the root element.
+- `heading_id`: A unique ID for your heading element. This will be used as the accessible name for the form.
+- `heading_tag`: The tag for your heading (defaults to `h2`).
+- `heading_text`: The text for your heading.
+- `heading_class`: An optional class for your heading.
```

Comment:
I was tempted to replace this all with a heading block,  but that makes it more likely for the `aria-labelledby` relationship between the form and heading to get messed up.